packages/http-client-csharp/emitter/test/Unit/emitter.test.ts

```diff
@@ -11,6 +11,7 @@ import { strictEqual } from "assert";
 import { statSync } from "fs";
 import { afterAll, beforeEach, describe, expect, it, Mock, vi } from "vitest";
 import { $onEmit, _validateDotNetSdk } from "../../src/emitter.js";
+import { CodeModel } from "../../src/index.js";
```

Comment:
Can we import this where it is defined? Importing from index.js will cause all of the exports to be imported.